Update docs related to selinux (#411)
apache/cloudstack-documentation: source/developersguide/ansible.rst

Save this as `/etc/ansible/roles/cloudstack-management/tasks/main.yml`

> .. note:: In a production environment, selinux would be set to enforcing

Reviewer: ```suggestion
.. note:: In a production environment, selinux should be set to enforcing
```

>    and the necessary selinux policies would be created to allow the

Reviewer: ```suggestion
   so that the necessary selinux policies are created to allow the
```
